### DISCUSSION TITLE
AddDaprComponent examples

### DISCUSSION THREAD

**Discussion by atrauzzi** [microsoft/aspire#4796] [Q&A] [answered]
Are there any more comprehensive examples of usages of `AddDaprComponent`?  Maybe particularly where we provide metadata?  And perhaps some explanation of when and why we'd use it?

Another related question... If I expect to use one implementation of say a secret store locally, but then a different one when deployed to the cloud, what should I do in my Aspire host application?  Do I detect my current environment (Production/Development) and then bind a different implementation to the service name?

Starting to have fun with this stuff...! :slightly_smiling_face:

**Comment by davidfowl** [comment, marked as answer]
The dapr implementation moved to https://github.com/CommunityToolkit/Aspire and there's lots of interesting work going on there around component that work both locally and in ACA.
